frontend/packages/dev-console/src/components/buildconfig/__tests__/BuildConfigFormPage.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '@console/internal/redux';
+import BuildConfigFormPage, { BuildConfigFormPageProps } from '../BuildConfigFormPage';
+
+jest.mock('../sections/EditorField', () =>
+  require.requireActual('@console/shared/src/components/formik-fields/TextAreaField'),
+);
+
+const Wrapper: React.FC<{}> = ({ children }) => <Provider store={store}>{children}</Provider>;
+
+describe('BuildConfigFormPage', () => {
+  it('should not fetch when creating a new BuildConfig', () => {
+    const props = {
+      match: { params: { ns: 'a-namespace', name: '~new' } },
+    } as BuildConfigFormPageProps;
+
+    render(
+      <Wrapper>
+        <BuildConfigFormPage {...props} />
+      </Wrapper>,
+    );
+  });
```

Comment:
there are no assertions here and below it blocks

Reply:
It was placeholders, but I updated them now with some different cases.